src/main/resources/l10n/Menu_da.properties

```diff
@@ -1,140 +1,136 @@
-#!
-#! created/edited by Popeye version 0.55 (github.com/JabRef/popeye)
-#! encoding:UTF-8
-
-Abbreviate_journal_names_(ISO)=Forkort_tidsskriftsnavn_(ISO)
-Abbreviate_journal_names_(MEDLINE)=Forkort_tidsskriftsnavn_(MEDLINE)
-About_JabRef=Om_&JabRef
-Append_library=Tilføj_ind&hold_fra_library
-Autogenerate_BibTeX_keys=&Autogenerer_BibTeX-nøgler
-Close_library=L&uk_library
+Abbreviate\ journal\ names\ (ISO)=Forkort tidsskriftsnavn (ISO)
```

Comment:
But now you replaced the underscores with backslashes. Compared to that I find underscores better